Sources/Misc/Signing.swift

```diff
@@ -178,6 +187,23 @@ extension Signing {
 
 }
 
+extension Signing.SignatureParameters {
+
+    var asData: Data {
+        // Not modified responses sign the Etag
+        let payload: Data = self.notModifiedResponse
```

Comment:
`payload` doesn't seem like a good name if this could have the etag. Don't know how to name it though... maybe `signatureMessage`? NABD though, I think it's easy enough to understand

Reply:
I agree 👍🏻  I copied this from Khepri but I'll rename it.